jupyterlab/labbox_ephys_widgets_jp/src/extensions/unitstable/Units/Units.tsx

```diff
@@ -52,21 +51,26 @@ const updateMetricData = (state: MetricDataState, action: MetricDataAction): Met
 
 type Label = string
 
-const Units: React.FunctionComponent<SortingViewProps> = (props) => {
-    const { sorting, recording, selectedUnitIds, onAddUnitLabel, onRemoveUnitLabel, onSelectedUnitIdsChanged, readOnly, sortingUnitMetrics, hither } = props
+interface OwnProps {
```

Comment:
Allow specifying a maxHeight. This is useful for when this widget is viewed inside SortingView as an accordion expandable pane. If there are hundreds of units, we don't want it to take up that much space in the accordion view.